src/main/resources/styles/reporting.css

```diff
@@ -57,6 +57,10 @@
 	background-color: #C5D88A;
 }
 
+.hidden {
+	visibility: hidden;
```

Comment:
There is no need for toggle, the passed,failed,pending, missing class will
be replaced with hidden only if the isHidden function is found true, as
there is no need to show something that is marked to be hidden.

In src/main/resources/styles/reporting.css
<https://github.com/masterthought/cucumber-reporting/pull/101#discussion_r30266049>
:

> @@ -57,6 +57,10 @@
>  	background-color: #C5D88A;
>  }
>
> +.hidden {
> +	visibility: hidden;

That sounds clear but I can't see in this PR where this class is enabled or
disabled. Is it new class or re-declared from existing one?

—
Reply to this email directly or view it on GitHub
<https://github.com/masterthought/cucumber-reporting/pull/101/files#r30266049>
.